# Robot pose (x, y, theta)
float64 x
float64 y
float64 theta

# Reference point gamma_s (x_ref, y_ref)
float64 gamma_x
float64 gamma_y

# CBF value and gradient
float64 h
float64 h_grad_x
float64 h_grad_y
